### ISSUE TITLE
Add the function to change the preview resolution of the camera

### ISSUE THREAD

**Issue by orange3134** [Yellow-Dog-Man/Resonite-Issues#129]
### Is your feature request related to a problem? Please describe.

The default camera available in Resonite is set to a preview resolution of 720p.
This is fine for taking pictures, but is problematic when taking videos.
If you shoot video at 1080p, you will get a blurry image.

### Describe the solution you'd like

Add an option to the camera to change the preview resolution.
1080p, 1440p, 2160p, etc. would be needed.

### Describe alternatives you've considered

Create a custom camera and add it to favorites.

However, there are situations where a custom camera does not work around this problem.
On some worlds, custom cameras are prohibited by permission settings.
To get a clean image in such worlds, I need to be able to change the preview resolution of the default camera.

**Comment by troyBORG** [comment]
When you say preview resolution.  Is that like what you see when you mirror to display?


I would think preview would be what it shows on back of the camera.

**Comment by orange3134** [comment]
@troyBORG Yes, the InteractiveCamera preview resolution matches the resolution of the image displayed on the screen in mirror to display.

**Comment by troyBORG** [comment]
Do you know where this is located, I'm sure in the way a mod can change to finger photo resolution.  We can find someone to make one that can change this automatically so it doesn't reset on restart.  Untill it's changed.

**Comment by orange3134** [comment]
@troyBORG It is one of the parameters of InteractiveCamera. If there is a mod that can change that, it would be helpful.

**Comment by troyBORG** [comment]
I tried editing the default camera and saved it and was able to set it as my custom camera.

I know when I spawned it for the streaming thing I did examine it, and it showed it saved the settings but still looks kinda weird when looking at text and stuff..  

When I play in desktop mode I swear things look alot sharper then when I mirror to the desktop.

I have the FXAA removal mod installed.
